pkgs/development/libraries/qxlsx/default.nix

```
@@ -17,6 +18,14 @@ stdenv.mkDerivation rec {
     hash = "sha256-twOlAiLE0v7+9nWo/Gd+oiKT1umL3UnG1Xa0zDG7u7s=";
   };
 
+  patches = [
+    # Fix for Qt 6.10, can likely be removed when version bump passes v1.5.0.
+    (fetchpatch {
+      url = "https://github.com/QtExcel/QXlsx/commit/90d762625750c6b2c73f6cd96b633e9158aed72e.patch";
```

```suggestion
      url = "https://github.com/QtExcel/QXlsx/commit/90d762625750c6b2c73f6cd96b633e9158aed72e.patch?full_index=1";
```
Future-proofing to prevent hash mismatch of the patch if and when the repo holds more commits. See [here](https://github.com/NixOS/nixpkgs/issues/266556#issuecomment-1837651068) for more info.

Ah, the `hash` will require updating after the change, I’m afraid.

---

I was checking to see if `fetchpatch` had been updated to forcibly include `?full_index=1` but that doesn’t seem to be the case.
Maybe a caching issue?

---

fetchpatch doesn't need the `full_index=1` parameter, because it strips away the line that normally changes depending on the number of commits in the repo (which full_index=1 can prevent).

---

I had to look it up -- it's `filterdiff` from patchutils that filters/normalizes/removes metadata from the patch before it gets hashed. So the `full_index=1` will have no effect on fetchpatch, AFAICT. (fetchpatch2 on the other hand benefits from `full_index=1`.)